### PR TITLE
Update euclid.

### PR DESCRIPTION
**Pull request by emilio**
This change is [Reviewable](https://reviewable.io/reviews/servo/skia/178)

### CODE REVIEW

**Comment by emilio** [comment]
The travis failure looks like a missing dependency in the Linux builders.

File: Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [package]
 
 name = "servo-skia"
-version = "0.30000021.2"
+version = "0.30000021.1"
```
Comment (jdm) [inline Cargo.toml]: .3, right?

Comment (emilio) [reply Cargo.toml]: Err, yes, this was a breaking bump, actually, but I had to add the fixup commit undoing it to keep fixing up servo, and then did the travis fixup on top. Fixed

**Pull request merged**